src/app/identity.service.ts

```diff
@@ -62,29 +62,44 @@ export class IdentityService {
   }
 
   getIdAndJWT(): { id: string, jwt: string } | undefined {
-    return this.storage.get(this.storageKey) || undefined;
+    const idAndJwt = this.storage.get(this.storageKey) || undefined;
```

Comment:
Is the `undefined` needed here? I presume it wouldn't if `this.storage.get(this.storageKey)` can return `null`.

Reply:
I think maybe not, let's try without